denizen_scripts/survival/repo_link/spawn_scripts/survival_rtp.dsc

```diff
@@ -27,7 +27,16 @@ survival_falloff_rtp:
   debug: false
   events:
     after player enters spawn_below:
+    - if !<player.flag[RecentRTP]>:
       - inject survival_rtp
+      - if <location[home_<player.uuid>]||null> != null:
+        - flag <player> RecentRTP duration:10m
+      - flag <player> RecentRTP duration:60s
+    - if <location[home_<player.uuid>]||null> != null:
+      - teleport <location[home_<player.uuid>]>
+      - narrate "<&a>You have been teleported home! Please wait <player.flag[RecentRTP].expiration.formatted> more before teleportation!"
+    - execute as_player "spawn"
+    - narrate "Please wait <player.flag[RecentRTP].expiration.formatted> seconds before teleporting again"
```

Comment:
This needs the `stop` command to prevent pieces of this script from running after picking a sector
Also, you should never `execute` a Denizen command - just run the script itself. The `spawn` command specifically only uses one line to teleport the player also, so we can just replace that with the teleport command.
```suggestion
    - if !<player.flag[RecentRTP]>:
      - inject survival_rtp
      - if <location[home_<player.uuid>]||null> != null:
        - flag <player> RecentRTP duration:10m
      - flag <player> RecentRTP duration:60s
      - stop

    - else if <location[home_<player.uuid>]||null> != null:
      - teleport <location[home_<player.uuid>]>
      - narrate "<&a>You have been teleported home! Please wait <player.flag[RecentRTP].expiration.formatted> more before teleportation!"
      - stop

    - teleport spawn
    - narrate "Please wait <player.flag[RecentRTP].expiration.formatted> seconds before teleporting again"
```